docs/user-guide/iam/ldap-configuration.md

```diff
@@ -119,3 +119,14 @@ After creating LDAP, you will see action buttons on the left side below the inpu
 ### **Delete LDAP**
 
     To permanently remove the LDAP settings, click Delete LDAP. A confirmation message will appear; confirm the action by selecting "Yes, delete" to finalize the removal.
+    
+### **Audit LDAP**
+
+To monitor LDAP operations, go to the `Settings` menu and select the `IAM Audit` tab under `IAM`. 
```

Comment:
ultra small nit: "to monitor" would be something I'd use to see what is happening right now, like live tracking the progress of something that is inflight. Retroactively looking I would call "Auditing"